admin/class-automation.php

```diff
@@ -59,35 +89,25 @@ class Automation {
 	 * Initialize functionality of this class by registering hooks.
 	 */
 	public static function init(): void {
-		add_action( 'admin_init', [ __CLASS__, 'action__admin_init' ] );
+		add_action( 'init', [ __CLASS__, 'action__init' ] );
 		add_action( 'admin_menu', [ __CLASS__, 'action__admin_menu' ] );
+		// Hook below called in self::render_submenu_page() instead.
+		// add_action( 'admin_print_scripts', [ __CLASS__, 'action__admin_print_scripts' ] );
 		add_filter( 'apple_news_article_metadata', [ __CLASS__, 'filter__apple_news_article_metadata' ], 0, 2 );
 	}
 
 	/**
 	 * A callback function for the admin_init action hook.
 	 */
-	public static function action__admin_init(): void {
-		add_settings_section(
-			self::PAGE_NAME,
-			esc_html__( 'Configuration', 'apple-news' ),
-			'__return_null',
-			self::PAGE_NAME
-		);
-		add_settings_field(
-			self::FIELD_NAME,
-			esc_html__( 'By Taxonomy Term', 'apple-news' ),
-			[ __CLASS__, 'render_settings_field' ],
-			self::PAGE_NAME,
-			self::PAGE_NAME
-		);
+	public static function action__init(): void {
 		register_setting(
 			self::PAGE_NAME,
 			self::OPTION_KEY,
 			[
 				'default'           => [],
 				'description'       => __( 'Automation settings for Publish to Apple News.', 'apple-news' ),
 				'sanitize_callback' => [ __CLASS__, 'sanitize_setting' ],
+				'show_in_rest'      => self::SCHEMA,
```

Comment:
Nitpicky, but it would be a bit cleaner if the schema constant contained the value of `schema` rather than having it be a top-level key, so it would have both `type` and `items` as its top level keys, and this would be changed to:
```suggestion
				'show_in_rest'      => [ 'schema' => self::SCHEMA ],
```